src/electrum_aionostr/relay.py

```diff
@@ -86,19 +99,20 @@ async def _receive_messages(self):
                     sys.stderr.write(message)
             except asyncio.CancelledError:
                 return
-            except exceptions.ConnectionClosed:
+            except client_exceptions.WSMessageTypeError:  #  raised by receive_json when connection is closed
```

Comment:
This type was only added in aiohttp 3.11
https://github.com/aio-libs/aiohttp/commit/fb7295489c537b509e886f6480bf94048b70c9c3

Reply:
I raised the min required version to that in https://github.com/spesmilo/electrum-aionostr/commit/2301e1f02b8722be8bf41f1dc5d5fce497109366